src/components/EditBook/EditBookEdition.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import Input from '../Input/Input';
+import { THUMBNAIL_DIMENSIONS } from '../../constants/books';
+
+const EditBookEdition = ({ currentBook }) => {
+  const [author, setAuthor] = useState(currentBook.author);
+  const [title, setTitle] = useState(currentBook.title);
+  const [description, setDescription] = useState(currentBook.description);
+  return (
+    <main>
```

Comment:
main is already used in Layout and you should not nest main in another main so it can be removed